airflow/models/skipmixin.py

```diff
@@ -98,3 +98,13 @@ def skip_all_except(self, ti: TaskInstance, branch_task_ids: Union[str, Iterable
 
             self.log.info("Skipping tasks %s", [t.task_id for t in skip_tasks])
             self.skip(dag_run, ti.execution_date, skip_tasks)
+
+    def evaluate_skip_condition(self, xcom_value, downstream_ti):
+        """
+        This method evalutes the xcom_value of the SkipMixin itself and decides if a downstream task
+        should be skipped. Returns True if the downstream_ti should be skipped. Otherwise False.
```

Comment:
You can add `:returns:` and `:rtype:` to the end of the docstring instead of putting it to the description.